catalogue/webapp/components/SignInBar/SignInBar.tsx

```diff
@@ -59,9 +59,29 @@ const Reload: FC<ReloadProps> = ({ reload }) => {
   );
 };
 
-const SignInBar: FC = () => {
+type Props = {
+  requestingUnavailable?: boolean;
+};
+
+const SignInBar: FC<Props> = ({ requestingUnavailable }) => {
   const { state, reload } = useUser();
 
+  if (requestingUnavailable) {
+    return (
+      <StyledComponent>
+        <Space h={{ size: 's', properties: ['margin-right'] }}>
+          <Icon icon={memberCard} />
+        </Space>
+        <AlignFont>
+          <span className={font('hnb', 5)}>Library members:</span>{' '}
+          <span className={font('hnr', 5)}>
+            Requesting is currently unavailable, while our building is closed.
```

Comment:
Minor: not required for this PR, but it'd be nice to have this in a more easily accessible variable/fetched from Prismic so the wording can be handled by the editorial team. I can imagine us reusing this if, say, we have more Auth0 or Sierra issues.